src/components/NavBar.tsx

```
@@ -4,9 +4,10 @@ import { motion } from "framer-motion";
 
 interface NavBarProps {
   onLogoClick?: () => void;
+  onToggleSidebar?: () => void;
 }
 
-export default function NavBar({ onLogoClick }: NavBarProps) {
+export default function NavBar({ onLogoClick, onToggleSidebar }: NavBarProps) {
```

ESLint

[error] 'onToggleSidebar' is defined but never used.

(@typescript-eslint/no-unused-vars)